openduck-py/openduck_py/routers/voice.py

```diff
@@ -412,3 +423,59 @@ async def audio_response(
     await responder.response_queue.join()
     await websocket.close()
     await log_event(db, session_id, "ended_session")
+
+
+async def connect_daily():
+    session_id = str(uuid4())
+
+    Daily.init()
+    mic = Daily.create_microphone_device(
+        "my-mic", sample_rate=OUTPUT_SAMPLE_RATE, channels=1, non_blocking=True
+    )
+    speaker = Daily.create_speaker_device(
+        "my-speaker", sample_rate=WS_SAMPLE_RATE, channels=1
+    )
+    Daily.select_speaker_device("my-speaker")
+    client = CallClient()
+    client.update_subscription_profiles(
+        {"base": {"camera": "unsubscribed", "microphone": "subscribed"}}
+    )
+    client.join(
+        meeting_url="https://matthewkennedy5.daily.co/Od7ecHzUW4knP6hS5bug",
+        client_settings={
+            "inputs": {
+                "camera": False,
+                "microphone": {
+                    "isEnabled": True,
+                    "settings": {"deviceId": "my-mic"},
+                },
+                "speaker": {
+                    "isEnabled": True,
+                    "settings": {"deviceId": "my-speaker"},
+                },
+            }
+        },
+    )
+    responder = ResponseAgent(
+        session_id=session_id, record=False, input_audio_format="int16"
+    )
+    asyncio.create_task(daily_consumer(responder.response_queue, mic))
+    while True:
+        if _check_for_exceptions(responder.response_task):
+            responder.audio_data = []
+            responder.response_task = None
+            responder.time_of_last_activity = time()
+
+        message = speaker.read_frames(WS_SAMPLE_RATE // 10)
```

Comment:
why // 10?

Reply:
WS_SAMPLE_RATE // 10 = 0.1 seconds, arbitrary chunk of time to process at once